connector.py

```diff
@@ -0,0 +1,24 @@
+from connectors.core.connector import Connector, ConnectorError, get_logger
+
+from .operations import *
+from .operations import _check_health
+
+logger = get_logger('dnstwist')
+
+
+class DnsTwist(Connector):
+    def execute(self, config, operation_name, params, **kwargs):
+        logger.info("operation_name: {0}".format(operation_name))
+        result = None
+        if operation_name == 'search':
```

Comment:
can we create an operations map for actions? This would avoid if else if more action is needed in future

        operations = {
            'search': search,
        }
        action = operations.get(operation)
        return action(config, params)